src/main/java/com/ssafy/sandbox/email/service/EmailScheduler.java

```diff
@@ -0,0 +1,27 @@
+package com.ssafy.sandbox.email.service;
+
+
+import com.ssafy.sandbox.email.repository.EmailVerificationRepository;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Slf4j
+@EnableScheduling
+@Service
+@RequiredArgsConstructor
+public class EmailScheduler {
+    private final EmailVerificationRepository emailVerificationRepository;
+
+    @Scheduled(fixedRate = 300000)
```

Comment:
```suggestion
    @Scheduled(fixedRate = 1000 * 60 * 5)
```
위와 같이 5분이라는 시간이 눈에 잘 들어오도록 명시하는 것도 괜찮을 것 같습니다!
다만 5분마다 해당 트랜잭션이 실행되는 것이 데이터 베이스에 부담이 될 것 같은데 이러한 부분을 고려하셨나요?

Reply:
그런 식으로 가독성을 개선할 수 있군요. 배워갑니다. 5분마다 삭제해주는 것이 조회 시간을 단축시켜주어 성능에 좋을 것이라고 생각했는데, 오히려 부담이 될까요? 그 점에 대해서 조금 더 고민해보겠습니다.

Reply:
아무래도 5분마다 인증 관련 테이블을 전체 탐색해서 해당 레코드의 expiry_date가 현재 시간보다 이전인지 확인하고 삭제하는 연산하는 전체 과정이다보니 데이터의 수가 늘어날 수록 부담이 증가할 수 있다는 생각이 들었습니다.